observations/obs_converters/HFradar/work/input.nml

```diff
@@ -0,0 +1,50 @@
+&preprocess_nml
+   overwrite_output        = .true. 
+   input_obs_qty_mod_file  = '../../../../assimilation_code/modules/observations/DEFAULT_obs_kind_mod.F90'
+   output_obs_qty_mod_file = '../../../../assimilation_code/modules/observations/obs_kind_mod.f90'
+   input_obs_def_mod_file  = '../../../forward_operators/DEFAULT_obs_def_mod.F90'
+   output_obs_def_mod_file = '../../../forward_operators/obs_def_mod.f90'
+   obs_type_files          = '../../../forward_operators/obs_def_ocean_mod.f90'
+   quantity_files          = '../../../../assimilation_code/modules/observations/ocean_quantities_mod.f90'
+  /
+
+
+../data/CODAR_BADA_2025_10_06_0310-1759720200.nc
+../data/BATU_2025_10_06_00_10_00_radial.nc
```

Comment:
@mgharamti think these lines (12 and 13) were left in accidentally and should be removed; they don't belong to any nml